templates/master.json

```diff
@@ -38,6 +48,14 @@
               "--v=2"
         ]
     }
-  ]
+  ],
+  "volumes": [
```

Comment:
this may be problematic on non-leader units... but lets scale test before we do anything in here